src/peft/tuners/lora/tp_layer.py

```diff
@@ -51,7 +51,7 @@ def __init__(
         LoraLayer.__init__(self, base_layer=base_layer)
 
         self.backend = backend
-        self.is_paralle_a = isinstance(base_layer, backend.RowParallelLinear)
+        self.is_parallel_a = isinstance(base_layer, backend.RowParallelLinear)
```

Comment:
Actually this is a breaking change - @szepeviktor would you mind opening a PR to re-add `self.is_paralle_a` as a property method with a FutureWarning for backward compatibility ?